---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* Ph.D. in Biomedical Sciences / Pharmacology, Icahn School of Medicine at Mount Sinai, 2025
  * Thesis: “Creating virtual twins of stem cell-derived cardiomyocytes to model variability in electrophysiology and drug response”
  * Thesis Advisor: Eric A. Sobie, PhD
* M.A. in Biomedical Informatics, Columbia University, 2020
  * Thesis: “Node embedding- and metapath-based features of a heterogenous biomedical information network for drug repurposing prediction”
  * Thesis Advisor: Nicholas P. Tatonetti, PhD
* B.A. in Cell Biology and Neuroscience, Rutgers University, 2016
  * Thesis: “Characterizing the interaction between alpha-catenin and Jub in Drosophila Melanogaster”
  * Thesis Advisor: Kenneth D. Irvine, PhD

Research interests
======
* Determining subcellular and environmental factors contributing to heterogeneity in disease phenotypes and drug responses, using computational systems biology models
* Integration of diverse datasets to extract novel insights into biological or engineering systems
* Optimization of prediction models for therapeutics outcome, adverse events, and disease
susceptibility

Research and work experience
======
* 2020-Present: Graduate Research Assistant
  * Icahn School of Medicine at Mount Sinai
  * Modeled variability in cardiac electrophysiology using the above cell-specific computational models, and predicting patient-specific drug-induced arrhythmia susceptibility
  * Skills: quantitative systems pharmacology, dynamical models, cardiac electrophysiology data analysis, MATLAB programming, parameter calibration, optimization algorithms
  * Supervisor: Dr. Eric A. Sobie

* 2019-2020: Graduate Research Assistant
  * Columbia University
  * Evaluated features and testing strategies for predicting drug treatments from a publicly available biomedical knowledge graph
  * Skills: Drug-drug and protein-protein interaction networks, graph embeddings, machine learning algorithms, Python programming, omics and sequencing data analysis
  * Supervisor: Dr. Nicholas P. Tatonetti

* 2016-2019: R&D Associate
  * Regeneron Pharmaceuticals, Inc.
  * Supported therapeutics discovery by performing antibody-antigen epitope mapping by hydrogen- deuterium exchange-mass spectrometry, using in-house-developed automated sample handling, liquid chromatography, and mass spectrometry methods
  * Optimized and troubleshot a biochemical enzymatic assay for initial compound library screening and characterization
  * Performed intact mass analysis in support of antibody characterization and de novo sequencing efforts
  * Prepared samples for proteomic, biochemical, and cell-based mass spectrometry experiments (IP, SDS-PAGE, trypsin/multienzyme digestion, tissue homogenization, cell feeding, etc.)
  * Audited and revised Investigational New Drug and Biologics License Application reports for Regeneron’s Scientific Writing Regulatory Liaison group
  * Supervisor: Dr. Yinyin Li
 
* 2013-2016: Undergraduate Research Assistant
  * Rutgers University, New Brunswick
  * Determined the mechanism through which alpha-catenin and Jub protein interaction alters Hippo pathway signaling and cell proliferation in Drosophila melanogaster
  * Skills: Protein quantification and Western blot, cell culture, histoimmunochemistry, confocal imaging, plasmid transfection, Drosophila genetics and maintenance
  * Supervisor: Dr. Kenneth D. Irvine

Skills
======
* Mathematical modeling and simulations of dynamical systems
* Proficient in MATLAB, Python, UNIX/shell scripting
* Experience with R, SQL
* Parallel computation on High Performance Computer systems
* Object-oriented programming, unit testing, and performance profiling 
* Machine learning, linear algebra, numerical integration
* Proficiency working with electrophysiology, proteomic, transcriptomic, and molecular structure data 
* Experience with fluorescence imaging and time series datasets
* Experience with X-ray imaging and medical record datasets

Teaching and leadership
======
* Teaching Assistant: Systems Biomedicine core course and lead TA for MATLAB bootcamp
* Mentorship: Assisted Sobie lab members and Icahn trainees with MATLAB coding and integration of
varied data modalities
* Leadership: Co-president of the Department of Pharmacological Sciences Student and Postdoc
Association: organized departmental events supporting career development, diversity, and inclusion

Fellowships and awards
======
* Traineeship, NICHD-Interdisciplinary Training in Systems and Developmental Biology and Birth Defects T32HD075735 (2021-Apr 2023)
* Fellowship, Department of Life Sciences Summer Undergraduate Research Fellowship (2015)
* Scholarship, School of Arts and Sciences Excellence Award (2014)
* Scholarship, Rutgers Presidential Scholarship (2012-2016)

Publications
======
* Combine the wet and dry ingredients: mixing experimental and computational models for a deeper understanding of stem cell-derived cardiomyocytes. Yang J, Sobie EA. In revision, *J. Physiol.* 2025
* Creating cell-specific computational models of stem cell-derived cardiomyocytes using optical experiments. Yang J, Daily NJ, Pullinger TK, Wakatsuki T, Sobie EA. *PLOS Computational Biology*. 2024 Sep 11.
* KEA3: Improved kinase enrichment analysis via data integration. Kuleshov MV, Xie Z, London ABK, Yang J, Evangelista JE, Lachmann A, Shu I, Torre D, Ma’ayan A. *Nucleic Acids Res*. 2021 Jul 2
* Epitope mapping by HDX-MS elucidates the surface coverage of antigens associated with high blocking efficiency of antibodies to birch pollen allergen. Zhang Q, Yang J, Bautista J, Badithe A, Olson W, Liu Y. *Anal Chem*. 2018 Aug 31
* Multiplex Immuno-Liquid Chromatography-Mass Spectrometry-Parallel Reaction Monitoring (LC-MS- PRM) Quantitation of CD8A, CD4, LAG3, PD1, PD-L1, and PD-L2 in Frozen Human Tissues. Zhang Q, Salzler R, Dore A, Yang J, Ma D, Olson W, Liu Y. J. *Proteome Res*. 2018 Oct 2
* Recruitment of Jub by alpha-catenin promotes Yki activity and Drosophila wing growth. Alégot H, Markosian C, Rauskolb C, Yang J, Kirichenko E, Wang Y-C, Irvine KD. J. *Cell Sci*. 2019 Feb 25
  
Talks
======
Oral Presentations
* ISMMS Department of Pharmacological Sciences Works-In-Progress (2023, 2024)
* ISMMS Department of Pharmacological Sciences Research Retreat (2023)
* IACM Conference: Mechanistic Machine Learning and Digital Engineering (2023)
* New York Academy of Sciences: Artificial Intelligence & Engineered Tissues For Cardiac Disease
Therapeutics (2022)

Posters
=====
* American Conference on Pharmacometrics (2024)
* American Heart Association Basic Cardiovascular Sciences Scientific Sessions (2024)
* Gordon Research Conference: Cardiac Arrhythmia Mechanisms (2023)
* Regeneron Science Forum (2018)
* Rutgers Cell Biology and Neuroscience Honors Symposium (2016)
* Rutgers Summer Undergraduate Research Fellows Symposium (2015)
  

